doc/cephadm/services/nfs.rst

```diff
@@ -49,9 +49,19 @@ Alternatively, an NFS service can be applied using a YAML specification.
         - host2
     spec:
       port: 12345
+      monitoring_port: 567
+      monitoring_ip_addrs:
+        host1: 10.0.0.123
+        host2: 10.0.0.124
+      monitoring_networks:
+      - 192.168.124.0/24
+
 
 In this example, we run the server on the non-default ``port`` of
 12345 (instead of the default 2049) on ``host1`` and ``host2``.
+Additionally, the default monitoring port can be changed using ``monitoring_port``,
+and the IP address for monitoring can be configured using ``monitoring_ip_addrs`` or
+IP of ``monitoring_networks``.
```

Comment:
I think we should be very explicit in the docs that the way this is implemented is that if we find a hostname match in `monitoring_ip_addrs` and if that ip is present on the host, we use that first. Then, if that doesn't happen, we will see if a network from `monitoring_networks` is present on the host and use an ip that matches that network if applicable. Lastly, we fall back to binding to all networks if neither of those work.